x-pack/plugin/ilm/src/main/java/org/elasticsearch/xpack/ilm/IlmHealthIndicatorService.java

```diff
@@ -62,11 +65,18 @@ public HealthIndicatorResult calculate(boolean includeDetails) {
                 Collections.emptyList()
             );
         } else if (ilmMetadata.getOperationMode() != OperationMode.RUNNING) {
+            List<HealthIndicatorImpact> impacts = Collections.singletonList(
+                new HealthIndicatorImpact(
+                    3,
+                    "Indices are not being rolled over, which could lead to future instability.",
+                    List.of(ImpactArea.SEARCH)
```

Comment:
Should we add a `DEPLOYMENT MANAGEMENT` impact area?

I wonder about the impact message. Rollover is an implementation detail. Maybe we shouldn't mention it in this case. 

How about?
```
Automatic index lifecycle and data retention management is disabled. The performance and stability of your system could be impacted. 
```